components/ledger/internal/bootstrap/config.rabbitmq.go

```
@@ -103,11 +109,13 @@ func initRabbitMQ(
 // initMultiTenantRabbitMQ initializes RabbitMQ in multi-tenant mode.
 // Uses tmrabbitmq.Manager for per-tenant vhost connections with LRU eviction.
 // No circuit breaker is needed; the Manager manages its own connection lifecycle.
+// redisConnection is required for Redis Pub/Sub event-driven tenant discovery.
 func initMultiTenantRabbitMQ(
```

golangci-lint (2.11.3)

[error] cognitive complexity 31 of func `initMultiTenantRabbitMQ` is high (> 30)

(gocognit)